hbase-client/src/main/java/org/apache/hadoop/hbase/client/HBaseAdmin.java

```diff
@@ -3159,7 +3159,7 @@ public QuotaRetriever getQuotaRetriever(final QuotaFilter filter) throws IOExcep
   @Override
   public List<QuotaSettings> getQuota(QuotaFilter filter) throws IOException {
     List<QuotaSettings> quotas = new ArrayList<>();
-    try (QuotaRetriever retriever = QuotaRetriever.open(conf, filter)) {
+    try (QuotaRetriever retriever = QuotaRetriever.open(connection, filter)) {
       Iterator<QuotaSettings> iterator = retriever.iterator();
```

Comment:
Changes looks good to me. 

However similar optimization can be done in master branch as well, below API is used multiple places
`org.apache.hadoop.hbase.quotas.QuotaRetriever#open(org.apache.hadoop.conf.Configuration, org.apache.hadoop.hbase.quotas.QuotaFilter)`

We can reuse the existing connection instead of creating new one always.

Reply:
Okay, I'm happy to expand the scope of this ticket.